tf_agents/agents/ppo/ppo_agent_test.py

```diff
@@ -69,7 +68,13 @@ def _get_normal_distribution_spec(self, sample_spec):
 
   def call(self, inputs, unused_step_type=None, network_state=()):
     hidden_state = tf.cast(tf.nest.flatten(inputs), tf.float32)[0]
-    batch_squash = network_utils.BatchSquash(self._outer_rank)
+
+    # Calls coming from agent.train() has a time dimension. Direct loss calls
+    # may not have a time dimension. It order to make BatchSquash work, we need
+    # to specify the outer dimension properly.
+    has_time_dim = hidden_state.ndim == 3
```

Comment:
Need tf 1.0 compatibility: 

```suggestion
    has_time_dim = tf.rank(hidden_state) == 3
```